cluster/manager_delete.go

```diff
@@ -129,14 +129,14 @@ func deleteUserNamespaces(kubeConfig []byte, logger *logrus.Entry) error {
 				continue
 			default:
 				logger.Infof("namespace %q still %s", ns.Name, ns.Status)
-				left++
+				left = append(left, ns.Name)
```

Comment:
Create structured log. See as an example how is done in `func (m *Manager) deleteCluster`.